src/main/java/it/gov/pagopa/payment/service/payment/idpaycode/IdpayCodeAuthPaymentServiceImpl.java

```diff
@@ -41,22 +42,22 @@ public AuthPaymentDTO authPayment(String trxId, String merchantId, PinBlockDTO p
         TransactionInProgress trx = idpayCodeAuthorizationExpiredService.findByTrxIdAndAuthorizationNotExpired(trxId);
 
         if(trx == null){
-            throw new ClientExceptionWithBody(HttpStatus.NOT_FOUND,
+            throw new TransactionNotFoundOrExpiredException(
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor